python/gui/auto_generated/qgsfilecontentsourcelineedit.sip.in

```diff
@@ -95,7 +95,52 @@ Emitted whenever the file source is changed in the widget.
 
 };
 
-class QgsSvgSourceLineEdit : QgsAbstractFileContentSourceLineEdit
+
+
+class QgsSvgOrImageSourceLineEdit : QgsAbstractFileContentSourceLineEdit
```

Comment:
This is a gross class name :stuck_out_tongue:  

How about QgsAbstractPictureSourceLineEdit or QgsPictureSourceLineEditBase?